tools/pipelines/templates/include-publish-npm-package.yml

```diff
@@ -47,10 +47,10 @@ stages:
       buildToolsVersionToInstall: ${{ parameters.buildToolsVersionToInstall }}
       pnpmStorePath: ${{ parameters.pnpmStorePath }}
 
-  # Only generate manifest files for runs in the internal project (i.e. CI runs, not PR run), for the main, test and release branch, and
+  # Only generate manifest files for runs in the internal project (i.e. CI runs, not PR run), for the main, release, legacy-break, and test branches, and
   # for the build of the client release group (we don't need manifests for anything else).
   # Enabling this template for every PR run risks overwriting existing manifest files uploaded to Azure blobs. Therefore, it's crucial to restrict this template to commits merged into the main and release branch.
-  - ${{ if and(eq(variables['System.TeamProject'], 'internal'), eq(parameters.tagName, 'client'), or(eq(variables['Build.SourceBranch'], 'refs/heads/main'), startsWith(variables['Build.SourceBranch'], 'refs/heads/release/'), startsWith(variables['Build.SourceBranch'], 'refs/heads/test/'))) }}:
+  - ${{ if and(eq(variables['System.TeamProject'], 'internal'), eq(parameters.tagName, 'client'), or(eq(variables['Build.SourceBranch'], 'refs/heads/main'), startsWith(variables['Build.SourceBranch'], 'refs/heads/release/'), startsWith(variables['Build.SourceBranch'], 'refs/heads/test/'), startsWith(variables['Build.SourceBranch'], 'refs/heads/legacy-breaks/client/'))) }}:
     - template: /tools/pipelines/templates/upload-dev-manifest.yml@self
```

Comment:
Why is the branch name not `test/legacy-breaks/client`? That might eliminate the need for this change.

Reply:
One concern I'd have about that is the fact that our test branches publish versions like `0.0.0-test-<buildId>` so consuming them might not be trivial, or in line with what our integration pipelines expect.

But also, I'm not sure how a new "kind" of branch (legacy-breaks) would behave in that regard.

Reply:
I broke apart from /test because we are looking at more automation we can provide for legacy-break work. I was not sure that everything that we build in for /test branches should apply to /legacy-breaks; there is overlap but these are different. And thought there should be clear representation that legacy-breaks are a thing.
`test/legacy-breaks/client/#.#0/foo` is also even longer, but only by 5 characters :)
I am okay starting with `test/legacy-breaks/client/#.#0/*`.